torch/CMakeLists.txt

```diff
@@ -476,15 +476,237 @@ install(FILES
     ${PROJECT_BINARY_DIR}/TorchConfig.cmake
     DESTINATION share/cmake/Torch)
 
-if (USE_DISTRIBUTED)
-  add_subdirectory(${TORCH_SRC_DIR}/lib/THD)
-  if (NOT MSVC AND NOT APPLE)
-    add_subdirectory(${TORCH_SRC_DIR}/lib/c10d)
+if (BUILD_PYTHON)
+  if (USE_DISTRIBUTED)
```

Comment:
we'd want to build distributed even when not building python right, i.e. for `libtorch`

Reply:
done (I guess the fact that all tests pass suggests a lack of CI coverage here - cc @teng-li @goldsborough)